src/schema/resolvers.js

```diff
@@ -1,3 +1,5 @@
+/* eslint no-underscore-dangle: [2, { "allow": ["_id"] }] */
```

Comment:
should we disable `no-underscore-dangle` in general?

Reply:
Ya if you want to. Although I think using underscore for naming convention is a bit weird looking.